### DISCUSSION TITLE
Charts in a shared Nightingale dashboard return an error?

### DISCUSSION THREAD

**Discussion by wknb** [ccfos/nightingale#1249] [Q&A]
夜莺版本：5.14.0
问题描述：点击监控大盘中的某个图标的分享按钮之后报错：Error 1364: Field 'cate' doesn't have a default value
request：/api/n9e/share-charts
response:
err:"Error 1364: Field 'cate' doesn't have a default value"

**Comment by UlricQin** [comment]
打开Chrome开发者工具，Request和response截图看看。前后端详细版本发一下。从系统信息页面拷贝一下。另外你是不是自行修改过表结构。执行这个命令看看结果：
grep cate a-n9e.sql
